cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandController.java

```diff
@@ -150,6 +156,7 @@ public DebeziumOperandController(KubernetesClient kubernetesClient, DebeziumOper
             .withMetadata(new ObjectMetaBuilder()
                 .withName(connector.getMetadata().getName())
                 .addToLabels(STRIMZI_DOMAIN + "cluster", connector.getMetadata().getName())
+                .addToLabels(BF2_DOMAIN + "connector.operator", operatorName)
```

Comment:
Looks like what we really use here is a label name (`BF_DOMAIN + connector.operator` in all cases. Maybe we can just use a single constant string instead of appending it multiple times?